.github/workflows/containers.yml

```diff
@@ -0,0 +1,87 @@
+name: Create containers
+
+on:
+  # run every night
+  schedule:
+    - cron: "0 22 * * *"
+
+  # schedule manually
+  workflow_dispatch:
+    inputs:
+      # On workflow dispatch, `branch` is selected by default
+      # You can access it in `github.ref_name`
+
+      tag_name:
+        description: "Tag name for the container"
+        required: true
```

Comment:
why is it required if there is a default?

Reply:
to have default value in the field